.storybook/grid/GridPanel.tsx

```diff
@@ -16,14 +16,31 @@ type Props = {
     api: API;
 };
 
+type CropAsset = {
+    file: URL;
+    size?: number;
+    mimeType?: string | number;
+    dimensions?: { width: number; height: number; };
+    secureUrl?: URL;
+};
+
+function getBestCrop(cropAssets: CropAsset[]) {
+    const IDEAL_WIDTH = 500;
```

Comment:
This is definitely an improvement on the previous behaviour of just picking the last crop in the list. Not for this PR but it'd be interesting to think about how we parameterise this when we want to use the image picker for other components which might have different requirements (I think we might want to use this on the newsletter epic).

Reply:
Would a comment be useful to explain how we've settled on 500?

Reply:
Yeah that's a great shout, partly because it would force me to remember the reasoning!

Reply:
This was a really great question to ask! Having spoken to folks more familiar with the Fastly image service than I am, it seems like we should always be using the `master` crop as the source (which makes sense, since Fastly is able to do the resizing, so we don't need to rely on the Grid to create different crop widths for us).